### PR TITLE
Task3

### PR DESCRIPTION
**Pull request by EiadAhmed**

### CODE REVIEW

**Review by amrbahaa1 — CHANGES_REQUESTED** [review]
Please add folder structure for scripts and styles

**Comment by EiadAhmed** [comment]
updated

File: scripts/home.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      temp += `</div>
+      </div>`;
+      ay.innerHTML = temp;
+    });
+}
+document.getElementById("search-form").addEventListener("submit", () => {
+  window.location.href += "?q=" + document.getElementById("search-text").value;
+  /* console.log(document.getElementById("search-text").value);*/
+});
+let a = location.href;
```
Comment (amrbahaa1) [inline scripts/home.js:64]: Variable names are not good. Please have a good naming variable.

**Review by amrbahaa1 — COMMENTED** [review]
Good job but try to always use a good naming convention

**Pull request merged**